### ISSUE TITLE
Player events do not dispatch with v0.2.1

### ISSUE THREAD

**Issue by jfairley** [clappr/clappr#521]
I've been trying to figure this out without success. When using the HLS playback, I can see that the playback is triggering events just fine, and the container is dispatching events just fine. Somewhere between the player and the container there is a breakdown, and the event handler / re-dispatcher functions inside the player never get triggered.

Specifically, [this](https://github.com/clappr/clappr/blob/0.2.1/src/components/container/container.js#L130) is called as expected.
```javascript
  timeUpdated(position, duration) {
    this.currentTime = position
    this.trigger(Events.CONTAINER_TIMEUPDATE, position, duration, this.name)
  }
```

However, [this](https://github.com/clappr/clappr/blob/0.2.1/src/components/player.js#L84) is never called.
```javascript
  onTimeUpdate(position, duration) {
    this.trigger(Events.PLAYER_TIMEUPDATE, position, duration)
  }
```

We depend on these events to control custom-built components, like our player's custom scrub bar. In the meantime, we will have to implement a `setInterval` function in our app to make up for no events. :frowning:

**Comment by jfairley** [comment]
FYI: @lkkadiri, @dviramontes

**Comment by towerz** [comment]
Indeed there was an error on the player event binding. Thanks @jfairley!

**Comment by jfairley** [comment]
Good find: https://github.com/clappr/clappr/commit/8d57f8417acd6693e2d98782df2fedffa461cc68

I tested locally, and it works great. Thanks!

When can we expect a 0.2.2?

**Comment by towerz** [comment]
I'm just adding some other improvements to hls, but I think I'll have it done today.

**Comment by towerz** [comment]
Hey @jfairley, I just released clappr 0.2.2 with this fix. Please let me know if you find anything else.

**Comment by jfairley** [comment]
:+1: looking great. thanks.
